### PR TITLE
Add UnitTests for #679

### PR DESCRIPTION
**Pull request by pquerner**
After #679 is now merged, this adds some unit tests for `Validate::is_ipv6`

### CODE REVIEW

**Comment by d00p** [comment]
Ok, and what about replacing the `filter_var` on IPv6 checking with the new `is_ipv6` function?

File: tests/Froxlor/ValidateTest.php [hunk 1]
```diff
@@ -109,13 +109,27 @@ public function testValidateCidrNoationToNetmaskNotationIPv4()
         $this->assertEquals("8.8.8.8/128.0.0.0", $result);
     }
 
+    public function testIPv6Checks()
```
Comment (d00p) [inline tests/Froxlor/ValidateTest.php]: please stay conform with the naming regarding the rest of the tests, ergo `testValidateIPv6` would be more suitable

**Comment by pquerner** [comment]
1 step at a time. First Unittests, then the world.

File: tests/Froxlor/ValidateTest.php (hunk 1 above)
Comment (pquerner) [reply tests/Froxlor/ValidateTest.php]: Changed

**Pull request merged**